src/olympia/devhub/templates/devhub/addons/edit/technical.html

```diff
@@ -40,34 +40,6 @@ <h3>
               {% endif %}
             </td>
           </tr>
-          <tr>
-            <th>
-              {{ tip(_("View Source?"),
-                     _("Whether the source of your add-on can be displayed in our "
-                       "online viewer. It is only relevant for listed add-ons.")) }}
-            </th>
-            <td>
-              {{ flags(_("This add-on's source code is publicly viewable."),
-                       addon.view_source if not editable else form.view_source, editable,
-                       _("No, this add-on's source code is not publicly viewable.")) }}
-            </td>
-          </tr>
-          {% endif %}
-          {% if show_listed_fields %}
-          <tr>
-            <th>
-              {{ tip(_("Public Stats?"),
-                     _("Whether the download and usage stats of your add-on can "
-                       "be displayed in our online viewer. "
-                       "It is only relevant for listed add-ons.")) }}
-            </th>
-            <td>
-              {{ flags(_("This add-on's stats are publicly viewable."),
-                       addon.public_stats if not editable else form.public_stats, editable,
-                       _("No, this add-on's stats are not publicly viewable.")) }}
-            </td>
-          </tr>
-          {% endif %}
```

Comment:
I think this place that be out of range about "View Source".

Reply:
I didn't quite get what you are implying? This section does not need to be deleted? 🤔

Reply:
You've deleted too much - the stats settings aren't going.

Reply:
I don't know if you meant it. I think that Area "Public Stats?" as well as area "View Source?" is included.
And, as @eviljeff  said that the template grammar is broken.